src/parentheses/Evaluation.java

```diff
@@ -0,0 +1,51 @@
+package parentheses;
+
+import java.util.Stack;
+
+public class Evaluation {
+
+    public boolean checkNextParentheses(Stack<Character> stack, char character, int i) {
```

Comment:
I named the different cases of the following switch statment code fragment A, B and C.
Those three fragments: A, B and C; do the same. They only differ in the `openCharacter` the evaluate.
Please create a method that receives that open character. Then use that method on each switch case.
This should be the final code:

```java
switch (character) {
    case ')':
        allOk = theNewMethod(stack, '(');
    case '}':
        allOk = theNewMethod(stack, '{');
    case ']':
        allOk = theNewMethod(stack, '[');
    default:
}
```